src/concrete/ml/onnx/ops_impl.py

```diff
@@ -893,11 +897,19 @@ def numpy_equal(
     Args:
         x (numpy.ndarray): Input tensor
         y (numpy.ndarray): Input tensor
+        lsbs_to_remove (Optional[int]): The number of the least significant bits to remove
 
     Returns:
         Tuple[numpy.ndarray]: Output tensor
     """
 
+    if lsbs_to_remove is not None and lsbs_to_remove > 0:
+        # opt 1 :R == D <=> R >= D
+        # opt 2: R == D <=> R > D - 1 (better)
+        return rounded_comparison(
+            y, x, lsbs_to_remove, operation=lambda x: x >= 0
+        )  # pragma: no cover
```

Comment:
I don't think this pragma no cover is right : the feature is tested right ?

Reply:
forgot to remove it.
It's tested.

Reply:
I still see the pragma 😅

Reply:
to use rounding in the second we need big datasets, with high n_bits
I mentioned it in my review.

Reply:
I posted a message in the PR, that has gone because I squashed some commits ...

> but is this something that will be fixed later ?

We would need to add a test with a big dataset, to enable rounding in the second stage.
I'll ask the team in the weakly if we agreed to add it, because it will slow down the CI.

Reply:
if it only happens in very uncommon and specific cases, which makes it not tested, maybe this feature is not worth then ? yes let's talk about it at the weekly !